summaries/analysis/Analyzer.py

```diff
@@ -11,18 +11,27 @@
 from rouge_score.rouge_scorer import _create_ngrams, _score_lcs, _score_ngrams
 from spacy.language import Language
 import matplotlib.pyplot as plt
+from datasets import Dataset
 from seaborn import histplot
 from tqdm import tqdm
 
 from ..utils import get_nlp_model, interpret_lang_code, find_closest_reference_matches
 
 
+valid_comparison_methods = ["exact"]
+
+
 class Analyzer:
     processor: Optional[Language]
     lang_code: Optional[str]
     lemmatize: bool
+    print_cutoff_length: Optional[int]
```

Comment:
Should be re-ordered to have `Optional`s after the definite arguments (i.e., `lemmatize` should be first).